jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -18,7 +18,7 @@ def buildExclusions = [
   [ /centos[67]-(arm)?(64|32)-gcc48/, anyType,     gte(10) ],
   [ /centos[67]-(arm)?(64|32)-gcc6/,  anyType,     lt(10)  ],
   [ /centos6-32-gcc6/,                releaseType, gte(10) ], // 32-bit linux for <10 only
-  [ /^centos7-64/,                    releaseType, lt(12)  ],
+  [ /^centos7-64/,                    anyType,     lt(12)  ],
```

Comment:
Can you explain the need for this a bit more.  Wondering if maybe the addition should be ppc specific

Reply:
In specific I'm wondering why we would not have already had the exclude for x64 in test.  Do we not have centos in the builds, that seems odd to me.

Reply:
https://github.com/nodejs/build/pull/1763#discussion_r276013923

> To start it should instead exclude running centos on versions lower than 12.x. The result should be builds on both ubuntu and centos for 12.x and above.

I am misunderstanding what you meant by that. What did you mean?

Reply:
maybe that applies only to centos7-ppc64? that version rule is wrong, we want the centos7-64 exclusion to only exist for `releaseType` because that's how we say that centos6 should be used for making binaries prior to 12. `anyType` is going to stop it even testing.

Reply:
My comment was because the tag is centos7-64 and I expected changes to only be affecting ppcle in this PR.